testsuite/src/it/java/org/graalvm/tests/integration/utils/BuildAndRunCmds.java

```diff
@@ -203,6 +206,12 @@ public enum BuildAndRunCmds {
                     new String[]{"powershell", "-c", "\"Expand-Archive -Path test_data.txt.zip -DestinationPath target -Force\""}
                     :
                     new String[]{"unzip", "test_data.txt.zip", "-d", "target"},
+            UsedVersion.getVersion(false).compareTo(Version.create(23, 1, 0)) >= 0 ?
```

Comment:
I don't think we can use `UsedVersion` like this. We'll have to add some tokens a la `JFR_MONITORING_SWITCH_TOKEN` and replace as appropriate for various versions. CI failures I think are also because of this, since we are hard-coding the used version to be non-container.

Reply:
Indeed